pkg/apis/scaler/v1alpha1/crd_demand.go

```diff
@@ -73,6 +73,12 @@ var (
 				Type:        "string",
 				JSONPath:    ".status.phase",
 				Description: "The phase of the Demand request",
+			}, {
+				Name:        "last transition time",
+				Type:        "string",
+				Format:      "date-time",
+				JSONPath:    ".status.last-transition-time",
+				Description: "The timestamp of the last phase transition of the Demand request",
```

Comment:
i would not include this in additional printer columns. This controls what is displayed when you run kubectl with `-o wide`. I think we already have enough info displayed with that.